From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michal Sojka <michal.sojka@cvut.cz>
Date: Sat, 28 Dec 2024 13:06:36 +0100
Subject: [PATCH] chore(autoware_pointcloud_preprocessor): Add missing
 dependencies

GMP is required in CMakeLists.txt. MPFR is required by GMP not
installed by default with GMP (Debian GMP package has MPFR dependency
only as a suggestion: https://packages.debian.org/sid/libgmp-dev).
---
 package.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package.xml b/package.xml
index faf4e9d383..2435cb0ee4 100644
--- a/package.xml
+++ b/package.xml
@@ -32,9 +32,11 @@
   <depend>cv_bridge</depend>
   <depend>diagnostic_updater</depend>
   <depend>image_transport</depend>
+  <depend>libgmp</depend>
   <depend>libopencv-dev</depend>
   <depend>libpcl-all-dev</depend>
   <depend>message_filters</depend>
+  <depend>mpfr</depend>
   <depend>nav_msgs</depend>
   <depend>pcl_conversions</depend>
   <depend>pcl_msgs</depend>
